e2e/test-headless-prompt-manager-ts/test/index.spec.ts

```diff
@@ -291,3 +291,41 @@ describe('AutoblocksPromptManager v1 weighted', () => {
     });
   });
 });
+
+describe('Undeployed', () => {
```

Comment:
this test is gonna fail until i go create an undeployed version in the CI instance